src/App.jsx

```diff
@@ -14,45 +16,99 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+const SORT_METHOD_ALPHABETICALLY = 'alphabetically';
+const SORT_METHOD_BY_LENGTH = 'length';
+
+const getViewGoods = (goods, sortMethod, isReversed) => {
+  const sortedGoods = [...goods];
+
+  switch (sortMethod) {
+    case SORT_METHOD_ALPHABETICALLY:
+      sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+
+    case SORT_METHOD_BY_LENGTH:
+      sortedGoods.sort((a, b) => a.length - b.length);
+      break;
+
+    default:
+      break;
+  }
+
+  if (isReversed) {
+    sortedGoods.reverse();
+  }
+
+  return sortedGoods;
+};
 
-    <ul>
-      <li data-cy="Good">Dumplings</li>
-      <li data-cy="Good">Carrot</li>
-      <li data-cy="Good">Eggs</li>
-      <li data-cy="Good">Ice cream</li>
-      <li data-cy="Good">Apple</li>
-      <li data-cy="Good">...</li>
-    </ul>
-  </div>
-);
+export const App = () => {
+  const [sortMethod, setSortMethod] = useState('');
+  const [isReversed, setIsReversed] = useState(false);
+  const viewGoods = getViewGoods(goodsFromServer, sortMethod, isReversed);
+
+  const resetSortingType = () => {
```

Comment:
the name of the function suggests that it will reset some type
but in fact it will reset sort method and turns off reverse
it's better to rename it to something abstract and common like `resetSettings`, `resetFilters`